layouts/partials/webinar-list-item.html

```diff
@@ -0,0 +1,21 @@
+<li class="w-full md:w-1/2 lg:w-1/3 m-0 p-2">
+    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col mx-auto" style="max-width: 320px;">
+        <div style="height: 239px;">
+            <div class="w-full h-full bg-contain bg-no-repeat bg-center bg-purple-500" style="background-image: url('{{ .Params.preview_image }}');"></div>
+        </div>
```

Comment:
It'd be nice if you could click on this image. Right now, you have to click below.

Reply:
👍